portfolio/src/main/webapp/index.html

```diff
@@ -1,16 +1,26 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="fullpage">
+		<div class="section">
+		</div>
+		<div class="section">
+		</div>
+        <div class="section">
```

Comment:
The indentation of these is misleading -- it suggests that maybe the first 2 `<div>`-s are somehow at a different level in the DOM three than the later 3.

Reply:
Indentation resolved with later commits 3d52c29c49bc4bb0319343aece1bae300b61a104, 9af3ca31c76c09be4cdd03c7bc2c351703e166f1, 3e3a7fb48fd3ebdceb9da199b8ca059eed448a23